filament/src/materials/ssao.mat

```diff
@@ -117,15 +117,30 @@ fragment {
         return (d >= -bias ? 0.0 : rangeCheck);
     }
 
+    float square(in float x) {
```

Comment:
This is not needed, we already have a function called `sq()` in the shaders to square a float.